### ISSUE TITLE
Uncaught Error: Cannot find module

### ISSUE THREAD

**Issue by antoniorhee** [electricjs/electric#65]
Hello, 

I'm running Electric on a Windows environment and have been seeing this error inside the Chrome console: 

Uncaught Error: Cannot find module "C:UsersliferayDesktopalloyui.com.temp askmetalsitelayoutslog.soy.js"

I've generated multiple different projects and continue to see this issue. This issue is keeping some of the JavaScript from running properly (e.g., Copy to Clipboard is not working). There seems to be an issue during the build process.  Here's a picture of the error message my console produces: 

![electric_cannotfindmoduleerror](https://user-images.githubusercontent.com/28714860/27407952-4c40b29c-568f-11e7-8ebb-04b0a3bd6b2d.jpg)

**Comment by Robert-Frampton** [comment]
Hey @antoniorhee thanks for reporting.

@blzaugg, have you ran into this at all on windows?

**Comment by blzaugg** [comment]
I have not. 

@antoniorhee, is this what you and @alexandervalencia asked me about?

I imagine the backslashes are not being escaped.

**Comment by blzaugg** [comment]
I take that back. I am getting that error.

**Comment by blzaugg** [comment]
```
__webpack_require__(!(function webpackMissingModule() { var e = new Error("Cannot find module \"D:liferaymetaljsmetaljs.comelectric.temp\taskmetalsitelayouts\blog.soy.js\""); e.code = 'MODULE_NOT_FOUND'; throw e; }()));
```

Not sure why `MODULE_NOT_FOUND`. But the path needs escaping in the error. Should be:

```
Error("Cannot find module \"D:\\liferay\\metaljs\\metaljs.com\\electric.temp\\taskmetalsitelayouts\\blog.soy.js\\"")
```

**Comment by antoniorhee** [comment]
Byran,

Yes this is the error that I asked you about before.

**Comment by Robert-Frampton** [comment]
Hey @antoniorhee 

I just published a new version that addresses this issue. Install the latest cli with `npm i -g electric-cli` and you should get the update.

Fixed at e7d044678d33def55f13b7b5b7c790c80b7cd6c7
